marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-common.json

```diff
@@ -12,6 +12,16 @@
       "action": "http://marklogic.com/xdmp/privileges/xdmp-eval-in",
       "kind": "execute"
     },
+    {
+      "privilege-name": "xdmp:invoke-in",
+      "action": "http://marklogic.com/xdmp/privileges/xdmp-invoke-in",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "xdbc:eval",
```

Comment:
This should take care of hub-central-user role getting 400 error on /project-info endpoint right?